### DISCUSSION TITLE
Page not rerendering on key change

### DISCUSSION THREAD

**Discussion by lazaroofarrill** [quasarframework/quasar#10677] [General - Components / Directives / etc] [answered]
I have a page which displays some entities from my database. I have the page key property bound to a store variable. When I update or delete something, I mutate the key, still the page doesn't update its content.

**Comment by metalsadman** [comment]
please provide a code snippet or better a codepen to show how you are doing it.

**Comment by ontwikkelfabriek** [comment]
>  I mutate the key , .... doesn't update ....

Sounds like:
https://vuejs.org/v2/guide/reactivity.html#Change-Detection-Caveats

**Comment by lazaroofarrill** [comment, marked as answer]
Two mistakes I was making. I was populating the view in the mounted hook but not in the update hook, and I was using the key in template tag instead of the page tag, though I'm not sure what that wouldn't work. Thanks anyway.
